### DISCUSSION TITLE
Still getting workflow failure notifications after not watching repo

### DISCUSSION THREAD

**Discussion by jsjoeio** [community/community#50508] [Actions]
### Select Topic Area

Bug

### Body

Hi! I used to maintain coder/code-server and set up a [scheduled workflow](https://github.com/coder/code-server/blob/main/.github/workflows/security.yaml#L11-L13) to run on `main` a while back.

I unsubscribed to the repo but still get notifications when the workflow failures. 
<img width="328" alt="image" src="https://user-images.githubusercontent.com/3806031/226389551-28955e60-f145-41d2-b672-a3f6a4705a5f.png">

[Example](https://github.com/coder/code-server/actions/runs/4469909136)

Is this a bug or is there something else I need to do to unsubscribe?

**Comment by airtower-luna** [comment]
The workflow you linked is marked as triggered by you, which would make you participating. I assume it's because [you're the last non-bot who edited the workflow file](https://github.com/coder/code-server/commits/a9d61daa915c104335267e0a17e9385b218a422b/.github/workflows/security.yaml). I haven't been in that situation myself, but my first try would be to ask one of the current developers to make a commit that changes the workflow file. :sweat_smile:

**Reply by jsjoeio** [reply]
LOL welp. I guess that's one way to do it. I will ping one of the current maintainers.

**Reply by jsjoeio** [reply]
🤞 https://github.com/coder/code-server/issues/6090

**Reply by jsjoeio** [reply]
Sadly it did not work. https://github.com/coder/code-server/commit/78282a1fd6f1dde0ff5867dfb3f3c489d48043d2

The current maintainer made a change to the file but I still got notified when one of the jobs failed

**Reply by jsjoeio** [reply]
<img width="1392" alt="image" src="https://user-images.githubusercontent.com/3806031/228007416-4d304732-62b9-4427-983d-ad5e24531716.png">

**Reply by airtower-luna** [reply]
Then I'm out of ideas, sorry. Maybe try [contacting GitHub support](https://support.github.com/request), someone stepping down from a project can't be such an unusual situation…

**Reply by jsjoeio** [reply]
Good idea. Trying there here: https://support.github.com/ticket/personal/0/2079619

**Comment by CodeMaster7000** [comment]


**Reply by jsjoeio** [reply]
That wouldn't help. I still want to be notified of actual projects I care about just not old ones.

**Comment by jsjoeio** [comment]
There are two ways to fix this according to GitHub Support:
1. edit the cron syntax in the workflow file
2. disable and re-enable the workflow

---

**Longer answer:**
> 
> The expected behavior for scheduled workflows in public repositories is that workflow notifications are sent to the user who initially created the workflow or the user who last edited the cron syntax in the workflow file.
> 
> https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/notifications-for-workflow-runs
> 
> The engineering team is currently working on a different approach in order to fix the issue for users who no longer belong to the repository.
> 
> In the meantime could you reach out to the team to confirm who the GITHUB ACTOR is on the workflow file? They can confirm this by running echo $GITHUB_ACTOR in your workflow file, If it is still set to your username, then they'll need to change it by either disabling and re-enabling the workflow or they'll need someone else to edit the cron section of the workflow file.
> 
> You mentioned the current maintainer made a change to the file but you still get notified when one of the jobs failed. Could you confirm if they made a change to the cron section of the file? They'll have to make a change there or either disable and re-enable the workflow.
> 
> See the guide below on how to disable and re-enable workflows:
> 
> https://docs.github.com/en/actions/managing-workflow-runs/disabling-and-enabling-a-workflow
